client/blocks/inline-help/index.jsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { identity } from 'lodash';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import debugFactory from 'debug';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal Dependencies
+ */
+import config from 'config';
+import { tracks } from 'lib/analytics';
```

Comment:
It's recommended to use the [analytics middleware](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics) instead of calling the analytics library directly. There's a basic example of the implementation [here](https://github.com/Automattic/wp-calypso/blob/4b1cde3f6a412505c0a1aee180abd61d6e9206d4/client/my-sites/people/people-invites/invites-list-end.jsx#L14).